### PR TITLE
make everything work in python 2.6

### PR DESCRIPTION
**Pull request by brettlangdon**
yeah, title says it all, now works in python 2.6, what I had to do was mostly, add backports of `assertIn`, `assertIsInstance` and `pydoc_data.topics`.

### CODE REVIEW

File: tests.py [hunk 1]
```diff
@@ -1,11 +1,61 @@
 import data
 import unittest
+import sys
 
 
-class DataTest(unittest.TestCase):
-    @classmethod
-    def setUpClass(cls):
-        cls.data = data.DATA
+class BackportTestCase(unittest.TestCase):
```
Comment (aaronchall) [inline tests.py:6]: just create these methods on the module level, then do something like:

``` python
if sys.version_info < (2, 7):
    unittest.TestCase.assertIn = assertIn
    unittest.TestCase.assertIsInstance = assertIsInstance
```

etc...

Comment (brettlangdon) [inline tests.py:6]: So the problem with something like this is that we are modifying the original class, this is mostly a no-no in OOP land, if we want to modify an existing class that we don't have control over the original definition we are better off creating a subclass and overwriting any definitions we want there. Make the OOP overlords happy :)

Comment (aaronchall) [inline tests.py:6]: ok, then do something like:

``` python
if sys.version_info < (2,7):
    TestCase = BackPortTestCase
else:
    TestCase = unittest.TestCase
```

and I'm ok with that.

Comment (brettlangdon) [inline tests.py:6]: @aaronchall updated the code

**Pull request merged**